pkg/restapi/restapi_test.go

```diff
@@ -20,7 +20,7 @@ import (
 func TestNew_Failure(t *testing.T) {
 	controller, err := New(&context.Provider{})
 	require.Error(t, err)
-	require.Equal(t, err, api.SvcErrNotFound)
+	require.Equal(t, err, api.ErrSvcNotFound)
```

Comment:
Should I just shorten to ErrNotFound?

Reply:
@troyronda in that case, question is how can we differentiate between different `NotFound` error cases (in same package)? e.g. 'Data not found' and 'Service not found'. Having a specific error message might help in future when we come across these type of scenario's imo.

Reply:
> Should I just shorten to ErrNotFound?

Yes.

Reply:
> @troyronda in that case, question is how can we going to differentiate between `NotFound` error cases (in same package)? e.g. 'Data not found' and 'Service not found'. Having a specific error message might help in future when we come across these type of scenario's imo.

`ErrSvcNotFound` is good for long run